### DISCUSSION TITLE
About App

### DISCUSSION THREAD

**Discussion by msalman360** [Koushikphy/Shared-Expense-Manager#4] [General]
Hi Sir, I have faced this type of error, if you have any idea kindly resolve it for me, thanks
error: 
FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':app:processDebugMainManifest'.
> Unable to make field private final java.lang.String java.io.File.path accessible: module java.base does not "opens java.io" to unnamed module @43bafc73

**Comment by msalman360** [comment]
I have facing this issue. kindly resolve 
 Where:
Script '/Users/apple/Desktop/development/flutter/packages/flutter_tools/gradle/flutter.gradle' line: 863

* What went wrong:
A problem occurred evaluating root project 'android'.
> A problem occurred configuring project ':app'.
   > Could not create task ':app:packLibsflutterBuildDebug'.
      > No signature of method: org.gradle.api.tasks.bundling.Jar.destinationDir() is applicable for argument types: (File) values: [/Users/apple/Desktop/development/Shared-Expense-Manager-master/build/app/intermediates/flutter/debug]

**Comment by Koushikphy** [comment]
I have not faced this problem. Maybe this could help https://stackoverflow.com/questions/75294879/no-signature-of-method-org-gradle-api-tasks-bundling-jar-destinationdir
